apps/dashboard/app/models/script.rb

```diff
@@ -193,12 +198,28 @@ def most_recent_job_cluster
     most_recent_job['cluster']
   end
 
+  def create_default_script
+    return if Script.scripts?(project_dir)
+    return if default_script_path.exist?
```

Comment:
Can you combine these into `return if Script.scripts?(project_dir) || default_script_path.exist?`. Little bit of a nit-pick, but I just think that's the ruby way (whatever that means).

Reply:
fixed